docs/onboarding/ck.hiring_process.how_to_guide.md

```diff
@@ -0,0 +1,216 @@
+- To see who is the HiringMeister now, refer to
+  [Rotation Meisters](https://docs.google.com/spreadsheets/d/1Ab6a3BVeLX1l1B3_A6rNY9pHRsofeoCw2ip2dkQ6SdA)
+- HiringMeister's duties include:
+  - Organizing the hiring and onboarding process (see [below](#step-by-step) for
+    more details)
+  - Supervising interns and people on a 3-month trial period
+  - Keeping all the relevant documentation up-to-date
+  - Improving the process when needed
+
+## Step by step
+
+- We look for candidates based on the specific hiring needs in our
+  teams/projects. There are several avenues:
+  - Emails to CS and related depts
+    - Especially for PhD students
+  - Using our LinkedIn pipeline
+    - We are going to start targeting graduates from top schools
+  - Job postings on LinkedIn
+  - Job postings on Upwork
+
+- HiringMeister: upon receipt of a new application, create a task on Asana in
+  the
+  [Hiring](https://app.asana.com/1/1208196877870190/project/1208280136292379/list/1208280159230261)
+  project
+  - The task goes in the "Candidates" group
+  - The name of the task is the name of the applicant
+  - In the task description, use the following template:
+
+    ```verbatim
+    Pronoun:
+    Personal email:
+    Work status (e.g., when graduates):
+
+    Hiring info:
+      Google Form:
+      CV:
+      LinkedIn:
+      GitHub user:
+      TG:
+      Working hours / week:
+      Best piece of code:
+      How good (1-5):
+    ```
+  - We use this Asana task to communicate about the applicant
+
+- HiringMeister: send an email to the applicant with a link to the
+  [questionnaire](https://docs.google.com/forms/d/e/1FAIpQLScWAavYiYj1IfWGP1QEv2jqjKvQKnFjseryhzmIIHZKnZ4HkA/viewform)
+  to gather information about them
+  - Responses are available
+    [here](https://docs.google.com/forms/d/1NUsjbTyXBoITQZ6z8nqmTydFPcWl0kWEQZJZzzYXUeM/edit#responses)
+  - Use their responses to update the corresponding Asana task description
+
+- Hiring team: decide whether we take on the applicant as an **intern**
+  - Decided at a review meeting that takes place every 2 weeks
+  - We want to increase the quality of the collaborators, so if there is a red
+    flag (e.g., no GitHub, low GPA, undergrad) we can decide not to offer the
+    internship
+  - The goal is to avoid onboarding people that will likely disappoint us
+  - If we receive a single candidate application and find no red flags in the
+    profile, we should proceed further in the process
+  - It's ok to ask more team members to take a look
+  - If the candidate is a no-go, GP sends an email of rejection
+
+- HiringMeister: create a GitHub issue for onboarding the intern
+  - Follow the instructions in
+    [`intern.onboarding_checklist.reference.md`](/docs/onboarding/intern.onboarding_checklist.reference.md)
+
+- HiringMeister: regularly check the updates made by the intern in the
+  onboarding issue and help resolve any errors they face
+  - The onboarding process should take 2-3 days max
```

Comment:
The "light" onboarding process ... and the goal is to make sure that all the mechanisms for developing are tested and working, so that we can focus on the next stages ("can the intern fix a simple bug?", "can the intern write more complex code?", etc)